### ISSUE TITLE
consistency issue with onSelect

### ISSUE THREAD

**Issue by Moezalez** [react-bootstrap/react-bootstrap#1268]
I have a problem with onSelect of Navbars NavItem vs MenuItem.

If I click through the items of this navbar:

```javascript
var Navigation = React.createClass({

  handleSelect: function(first, second){
    console.log(first, second);
  },

  render: function() {
    return (
      <Navbar brand='Navbar' toggleNavKey={0}>
        <CollapsibleNav eventKey={0} onSelect={this.handleSelect}>

          <Nav navbar>
            <NavItem eventKey={1}>Home</NavItem>
          </Nav> 


          <Nav navbar right hide>
            <NavItem eventKey={2}>Login</NavItem>

            <NavDropdown eventKey={3} title='NavDropdown' id='basic-nav-dropdown'>
              <MenuItem eventKey={4}>Action 1</MenuItem>
              <MenuItem eventKey={5}>Action 2</MenuItem>
            </NavDropdown>
          </Nav>

        </CollapsibleNav>
      </Navbar>

    )
  }
});
```

This is the output:
```
1 undefined
2 undefined
SyntheticMouseEvent 4
```

making it really hard to handle the selected item.

Version:  react-bootstrap@0.25.100-react-pre.0

**Comment by jquense** [comment]
Thanks for the heads up. We are working towards making these events consistent, and you caught us in the awkward middle phase.

**Comment by AlexKVal** [comment]
Closed by #1604
